### ISSUE TITLE
Add MAPPED_TIMESTAMP as a flag required for zeCommandListAppendQueryKernelTimestamps

### ISSUE THREAD

**Issue by joshuaranjan** [oneapi-src/level-zero-spec#140]
Currently for zeCommandListAppendQueryKernelTimestamps, the spec mentions

The application must ensure the events were created from an event pool that was created using _ZE_EVENT_POOL_FLAG_KERNEL_TIMESTAMP_

May be ZE_EVENT_POOL_FLAG_KERNEL_ **MAPPED** _TIMESTAMP has to be added like 

"The application must ensure the events were created from an event pool that was created using ZE_EVENT_POOL_FLAG_KERNEL_TIMESTAMP or ZE_EVENT_POOL_FLAG_KERNEL_MAPPED_TIMESTAMP"

**Comment by jandres742** [comment]
thanks @joshuaranjan . Yes, you are right.

**Comment by aravindksg** [comment]
agree, this seems natural to add to this API, but different question- is there a scenario where user's are interested to read timestamp values only on the device?
